library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -33,7 +39,7 @@
 import static com.novoda.downloadmanager.DownloadBatchStatus.Status.ERROR;
 import static com.novoda.downloadmanager.DownloadBatchStatus.Status.PAUSED;
 
-@SuppressWarnings("PMD.ExcessiveImports")
+@SuppressWarnings({"PMD.ExcessiveImports", "PMD.GodClass"})
```

Comment:
not ideal but the change to fix this would be bigger than the workmanager migration itself 😅